### DISCUSSION TITLE
BrowserShot fails when javascript has an alert

### DISCUSSION THREAD

**Discussion by moham96** [spatie/browsershot#818] [Bugs]
Hello,
I discovered this while using `laravel-query-detector` which causes an alert to be opened when it detects lazy loading
It seems like having an `alert` in the page causes browsershot to fail

For testing a simple script will show the issue

```php
$header = [
            'Content-Type' => 'application/pdf',
            'Content-Disposition' => 'inline; filename="Somefile.pdf"'
        ];
        $pdf = Browsershot::html("<script>alert('test')</script>");
        return response($pdf->pdf(), headers: $header);
```

which causes a timeout:
```
================ Error Output: ================ TimeoutError: Navigation timeout of 30000 ms exceeded at new Deferred 
```

**Comment by gdandersson** [comment]
dismissDialogs() now exists, which should solve the problem
